assets/get-started-sidebar.js

```diff
@@ -20,6 +24,34 @@ $("#windows").on("click", function() {
   showSidebar("windows");
 });
 
+$("#alibaba").on("click", function() {
+  showCloudSidebar();
+});
+
+$("#microsoft-azure").on("click", function() {
+  if ($(this).parent().hasClass('open')) {
+    showCloudSidebar();
+  }else{
+    showCloudSidebar("microsoft-azure");
+  }
```

Comment:
It looks like this block of code is being repeated a lot, and you're just changing the argument that gets passed to `showCloudSidebar`. You can probably make a new function that executes this block of code and just pass in the argument to the new function. There should also be spaces between `}else{`.